code/modules/unit_tests/nuke_cinematic.dm

```diff
@@ -3,6 +3,8 @@
 #define PLAYING_INCORRECT_NUKE_ANIMATION 1
 #define NOT_PLAYING_ANIMATION 0
 
+/** VOIDCREW REMOVAL
```

Comment:
lmao why do we need to remove this

Reply:
probably dont anymore, there are zlevels that are considered station now